src/swupd/README.vendor

```diff
@@ -0,0 +1,36 @@
+This package uses "dep" tool for vendoring packages, i.e. downloading
+the required files to use the package and storing it in the vendor/
+subdirectory.
+
+Note that this vendoring approach don't depend on git submodules or
+extra clones, the files are stored in the mixer repository. That way
+using "dep" is only needed when updating dependencies.
+
+The tool keeps track of the versions in Gopkg.toml (editable by the
+user) and Gopkg.lock (always auto-generated) files.
+
+To download/update dep, use
+
+    $ go get -u github.com/golang/dep/cmd/dep
+
+To add a new dependency, use a blank import somewhere in the code to
+let dep tool know that we need that dependency
+
+    import _ "github.com/ulikunitz/xz"
+
+and then
+
+    $ cd src/swupd
+    $ GOPATH=$PWD/../.. dep ensure -add github.com/ulikunitz/xz@v0.5.4
```

Comment:
Is this always needed even if they have GOPATH exported already?

Reply:
If the code is already inside a GOPATH, not anymore :-)

I'll fix these instructions.

Reply:
Hopefully it does. The next release will update the package structure and it will be a regularly go get-able project, which will automatically put it in users' GOPATH.

Reply:
Actually, until we rebase the branch doing the swupd work, these instructions are correct, so I'm not changing then.

Reply:
Ok, let's just not let it get swept away with the tons of other changes going in. It's not technically wrong to leave it though if they're already working in their GOPATH.